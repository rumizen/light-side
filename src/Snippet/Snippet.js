import React from 'react';
import './Snippet.scss';

const Snippet = ({ category, text }) => {
  return (
    <article className="snippet" id={Date.now()} key={Date.now()}>
      <h3>{category}</h3>
      <p>{text}</p>
    </article>
  );
};

export default Snippet;