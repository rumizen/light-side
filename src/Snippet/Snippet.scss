.snippet {
  border-bottom: solid lightgray 1px;
  width: 90%;
}