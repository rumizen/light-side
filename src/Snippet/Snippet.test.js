import React from "react";
import { shallow } from "enzyme";
import Snippet from "./Snippet";

describe("Snippet", () => {
  it("should match the Snippet snapshot", () => {
    const wrapper = shallow(
      <Snippet
        category="name"
        text="Luke"
      />);

    expect(wrapper).toMatchSnapshot();
  });
});
