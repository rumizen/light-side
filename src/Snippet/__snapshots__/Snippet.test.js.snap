// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snippet should match the Snippet snapshot 1`] = `ShallowWrapper {}`;
